.github/workflows/test_latest_versions.yml

```diff
@@ -49,6 +49,6 @@ jobs:
           notebook_flags=""
           if [ "$pver" = "3.12" ]; then
             echo Skipping tutorials that require cplex
-            notebook_flags="${notebook_flags} --ignore=docs/circuit_cutting/cutqc/tutorials/tutorial_1_automatic_cut_finding.ipynb"
```

Comment:
The previous line (line 51) should be removed, too.  And come to think of it, we can get rid of this whole `notebook_flags` business now, too.